foodmart-dataset/src/main/resources/druid/foodmart/foodmart-index.json

```diff
@@ -140,7 +140,7 @@
         },
         {
           "name" : "user_unique",
-          "type" : "hyperUnique",
```

Comment:
Might want to make a new column and keep the `hyperUnique` column. That way both can be tested.